src/lib/core/CHIPError.h

```diff
@@ -841,13 +841,21 @@ using CHIP_ERROR = ::chip::ChipError;
  *  @def CHIP_ERROR_IN_USE
  *
  *  @brief
- *    A value is already used. Generally indicates an unavailable resource.
+ *    A value is already used. Generally indicates an unavailable resrouce.
```

Comment:
```suggestion
 *    A value is already used. Generally indicates an unavailable resource.
```
Why this broken change?